design/adr/core/0002-Developer-Scripts-TaggingAndCleanup.md

```diff
@@ -0,0 +1,105 @@
+    [init_shell.bat](https://github.com/edgexfoundry/developer-scripts/blob/master/init_shell.bat),
+    [startdb-no-auth.bat](https://github.com/edgexfoundry/developer-scripts/blob/master/startdb-no-auth.bat),
+    [startdb.bat](https://github.com/edgexfoundry/developer-scripts/blob/master/startdb.bat),
+    [init_mongo.js](https://github.com/edgexfoundry/developer-scripts/blob/master/init_mongo.js),
+    [linux_setup.sh](https://github.com/edgexfoundry/developer-scripts/blob/master/linux_setup.sh) - **to be Removed**. <br /> 
+ All these scripts set up mongo database. Instead, go application [docker-edgex-mongo](https://github.com/edgexfoundry/docker-edgex-mongo) is supposed to be used.
+* [clean_mongo.js](https://github.com/edgexfoundry/developer-scripts/blob/master/clean_mongo.js)  - **to be Removed** <br />
+These file should be removed once a second executable is added in [docker-edgex-mongo](https://github.com/edgexfoundry/docker-edgex-mongo). Currently `docker-edgex-mongo` is single purpose application responsible for initializing mongo database (setting up users/passwords/privileges and creates databases and collections). In addition, executable for `purging` the entire database need to be added. It is supposed to work in both security enabled/disabled environments.
+It could be used by Edgex CLI for purge the entire database (without the need to have all the microservices up and running). More about this proposal [here](0002-MongoDB-Clearance.md)  
+
+* mongoAdminOperStateUpdate.js - **to be Removed** <br />
+The script is not relevant anymore
+
+* [prepare-environment.sh](https://github.com/edgexfoundry/developer-scripts/blob/master/prepare-environment.sh) -  **to be Modified** <br />
+This script clone/update all or set of modules (based on `MODULES`) directly from `https://github.com/edgexfoundry`. 
+    - add ability to clone both - main repository or developers forks. Usually developers work with `fork branches`. They do not modify the real repo by itself. 
+
+* [reset-dockers.sh](https://github.com/edgexfoundry/developer-scripts/blob/master/reset-dockers.sh) - **To be Optimized** <br />
+Removes all containers and images from working environment. Useful for development, when reset everything from scratch is needed. 
+    - Add optional parameter to specify compose file different from the default one.
+    - If docker file does not exists in the current directory - script should exit with error 
+    - Clean the volumes as well
+    - make the script more efficient by using other commands
+
+* [run-it.sh](https://github.com/edgexfoundry/developer-scripts/blob/master/run-it.sh) - **to be Removed**. 
+    - instead of using/maintaining this script the suggestion id to use: `docker-compose -f docker-compose-file-name.yml up -d`.
+
+* [update-packages.sh](https://github.com/edgexfoundry/developer-scripts/blob/master/update-packages.sh) -  **to be Modified** <br />
```

Comment:
Can we change the name to something more applicable like `pull-latest-modules`?

Reply:
I would suggest to even remove this script and stay only with the `prepare_environment.sh` / `clone-modules`  script ... however we call it
If not, I would suggest to call this script `pull_modules` ...but this will make the proposed name  `clone_modules` even more misleading

Reply:
Agree with remove. :-)